components/Article/PayNote.js

```diff
@@ -229,63 +301,62 @@ const TryNoteCta = compose(withRouter)(({ router, darkMode, payload }) => {
   )
 })
 
-const PayNoteCta = ({ variation, payload, position, darkMode }) => {
-  return (
+const PayNoteCta = ({ payNote, payload, darkMode }) =>
+  payNote.cta ? (
     <div {...styles.cta}>
-      {variation.cta === 'try' ? (
+      {payNote.cta === 'trialForm' ? (
         <TryNoteCta darkMode={darkMode} payload={payload} />
       ) : (
-        <BuyNoteCta variation={variation} position={position} />
+        <BuyNoteCta payNote={payNote} payload={payload} />
       )}
     </div>
-  )
-}
+  ) : null
 
 export const PayNote = compose(
   withRouter,
   withInNativeApp,
-  withMemberStatus
+  withMemberStatus,
+  graphql(memberShipQuery)
 )(
   ({
     router: { query },
     inNativeIOSApp,
     isEligibleForTrial,
+    hasActiveMembership,
+    data: { membershipStats },
     seed,
-    trial,
+    tryOrBuy,
     documentId,
     repoId,
-    series,
-    position
+    position,
+    customPayNotes
   }) => {
-    const variation = getPayNoteVariation({
+    const target = {
```

Comment:
One naming suggestion:
```suggestion
    const subject = {
```
`subject` meeting/matching `paynote.target` would make the code easier to follow.